src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -33,15 +33,17 @@ interface OutputInterface
     /**
      * Writes a message to the output.
      *
-     * @param $newline Whether to add a newline
-     * @param $options A bitmask of options (one of the OUTPUT or VERBOSITY constants), 0 is considered the same as self::OUTPUT_NORMAL | self::VERBOSITY_NORMAL
+     * @param bool                                          $newline Whether to add a newline
+     * @param int-mask-of<self::VERBOSITY_*|self::OUTPUT_*> $options A bitmask of options (one of the OUTPUT or VERBOSITY constants),
```

Comment:
We can also use the `self::VERBOSITY_*` literal type in `getVerbosity()`, `setVerbosity()` and the constructor of `Symfony\Component\Console\Output\Output`.

Reply:
Agreed, for the same reasons as on your first comment.